edgelet/edgelet-settings/src/base/mod.rs

```diff
@@ -68,6 +70,10 @@ pub struct Settings<ModuleConfig> {
     #[serde(default, skip_serializing)]
     #[cfg_attr(not(debug_assertions), serde(skip_deserializing))]
     pub endpoints: aziot::Endpoints,
+
+    /// Additional system information
+    #[serde(default)]
```

Comment:
Perhaps add `skip_serializing_if = "BTreeMap::is_empty"` so you don't have to add an empty [additional_info] table to all the test toml files.

Reply:
4634f903eae3a3f0782f1e4f66e46f417b2b2d1f